docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an
+argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `scheduler.alpha.kubernetes.io/name` annotation on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+
+Please see the
+[kube-scheduler documentation](http://kubernetes.io/docs/admin/kube-scheduler/) for
+detailed description of other command line arguments.
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment
+specified in the config above in a Kubernetes cluster:
+
+```shell
+$ kubectl create -f my-scheduler.yaml
+```
+
+Verify that the scheduler pod is running:
+
+```shell
+$ kubectl get pods --namespace=kube-system
+NAME                                           READY     STATUS    RESTARTS   AGE
+....
+my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
+...
+```
+
+You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
+pod in this list.
+
+### 4. Specify schedulers for pods
+
+Now that our second scheduler is running, let's create some pods, and direct them to be scheduled by either the default scheduler or the one we just deployed. In order to schedule a given pod using a specific scheduler, we specify the name of the
+scheduler as an annotation in that pod spec. Let's look at three examples.
+
+
+1. Pod spec without any scheduler annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: no-annotation
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-no-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  When no scheduler annotation is supplied, the pod is automatically scheduled using the
+  default-scheduler.
+
+  Save this file as `pod1.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod1.yaml
```

Comment:
For examples where it's just a simple command, leave the "$" out. For examples where it's a mix of the command and output from the command, leave the "$" in.